rskj-core/src/main/resources/reference.conf

```diff
@@ -53,6 +53,7 @@ blockchain = {
             rskip199 = iris300
             rskip200 = iris300
             rskip201 = iris300
+            rskip203 = arrowhead600
```

Comment:
Could you also add this RSKIP in ActivationConfigsForTest::getArrowhead600Rskips?

Reply:
Done